scripts/apps/authoring/views/authoring-topbar.html

```diff
@@ -77,7 +78,8 @@
                 <span class="btn__text--short" translate>P</span>
         </button>
         <button
-                ng-if="$root.config.features.customAuthoringTopbar.publishAndContinue === true && canPublishOnDesk()"
+                ng-if="$root.config.features.customAuthoringTopbar.publishAndContinue === true
+                && canPublishOnDesk() && publishAndContinueAvailable"
```

Comment:
would be good to have that logic in one place and here just check `ng-if="publishAndContinueEnabled"` otherwise you have to check both places to figure out what is going on